modules/ding_provider/ding_provider.exceptions.inc

```diff
@@ -80,13 +80,3 @@ class DingProviderUserException extends Exception {
     }
   }
 }
-
```

Comment:
Never have been. It provides no functionality, it has a private property nobody uses, and the only one using it was ding_user catching it.

But ding_user should catch any exception, and as nobody is throwing it anyway, I removed it.